app/Models/Post.php

```diff
@@ -9,13 +9,14 @@ class Post extends Model
 {
     use HasFactory;
 
-
+    protected $guarded = [];
+    
     public function getUserRelation()
     {
         return $this->belongsT0(User::class);
```

Comment:
This belongsTo is small letter 'o'na

Reply:
And since its a relationship, you dont need to use getUserRelationship. u can just name it user